library/alloc/src/boxed.rs

```diff
@@ -119,7 +119,6 @@
 //!
 //! [ucg#198]: https://github.com/rust-lang/unsafe-code-guidelines/issues/198
 //! [dereferencing]: core::ops::Deref
-//! [`Box<T>`]: Box
 //! [`Box::<T>::from_raw(value)`]: Box::from_raw
 //! [`Box::<T>::into_raw`]: Box::into_raw
```

Comment:
Your script missed this ;)

Reply:
Oh yes, I forgot about that kind! I guess I'll have to write a parser now ;)

Reply:
Actually I updated my script and it's able to catch that! (with some false positives of course)